private/bufpkg/bufconfig/generate_plugin_config.go

```diff
@@ -22,17 +22,10 @@ import (
 	"strings"
 
 	"github.com/bufbuild/buf/private/bufpkg/bufplugin/bufpluginref"
-	"github.com/bufbuild/buf/private/bufpkg/bufremoteplugin"
 	"github.com/bufbuild/buf/private/pkg/encoding"
 	"github.com/bufbuild/buf/private/pkg/syserror"
 )
 
-const (
-	remoteAlphaPluginDeprecationMessage = "the remote field no longer works as " +
```

Comment:
It's been >1 year since we stopped supporting legacy (alpha) remote plugins on the CLI and server side. So this is safe to drop now.